Shared/ViewModels/LatestMediaViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+/*
+ * SwiftFin is subject to the terms of the Mozilla Public
+ * License, v2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2021 Aiden Vigue & Jellyfin Contributors
+ */
+
+import Combine
+import Foundation
+import JellyfinAPI
+
+final class LatestMediaViewModel: ViewModel {
+    @Published
+    var items = [BaseItemDto]()
+
+    var libraryID: String
+
+    init(libraryID: String) {
+        self.libraryID = libraryID
+        super.init()
+        
+        refresh()
+    }
+
+    func refresh() {
```

Comment:
please don't name these functions `refresh`
name them something descriptive.
for example, this could be `getLatestMediaInLibrary`